eras/conway/impl/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1.3.0.1
 
-*
+* Add `Arbitrary` instances for `ConwayCertPredFailure`, `ConwayVDelPredFailure`, and `ConwayDelegPredFailure`
```

Comment:
Addition of instances constitute a minor version bump, so please change version to `1.3.1.0`:
* in the changelog section above
* and in the cabal file

Also, `testlib` changes require addition of special section:

```suggestion
### `testlib`

* Add `Arbitrary` instances for `ConwayCertPredFailure`, `ConwayVDelPredFailure`, and `ConwayDelegPredFailure`
```

Reply:
sorry, I think I still misunderstood the changelog process for ledger, even after your comments on the last PR 🫥

it should be fixed now!

Reply:
Ah, yes. `cardano-ledegr-conway` has a version bump just about an hour ago or so :smile: so changes in this PR no longer required to do anything about versions. 

Very nice. Thank you again.